### PR TITLE
Boarding pass

### PR DESCRIPTION
**Pull request by Roluisker**

### CODE REVIEW

File: app/src/main/res/layout/fragment_boarding_pass.xml [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+                    />
+
+        </RelativeLayout>
+
+        <RelativeLayout
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_weight="1">
+
+            <ImageView
```
Comment (feresr) [inline app/src/main/res/layout/fragment_boarding_pass.xml]: A RelativeLayout with a single view inside, we might be able to get rid of it. :)

File: app/src/main/res/layout/fragment_boarding_pass.xml [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+                    android:layout_marginTop="5dp"
+                    android:id="@+id/imageView"
+                    android:src="@drawable/common_signin_btn_text_focus_dark"/>
+
+            <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:textAppearance="?android:attr/textAppearanceSmall"
+                    android:text="Small Text"
+                    android:layout_marginLeft="10dp"
```
Comment (feresr) [inline app/src/main/res/layout/fragment_boarding_pass.xml]: Please remove all margins & paddings from XML. will be added latter with syles

File: app/src/main/res/layout/fragment_boarding_pass.xml [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+                    android:src="@drawable/abc_btn_check_to_on_mtrl_015"
+                    android:id="@+id/boardingPassChecked" />
+
+    </LinearLayout>
+
+    <LinearLayout
+            android:layout_width="fill_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            android:layout_marginTop="5dp"
```
Comment (feresr) [inline app/src/main/res/layout/fragment_boarding_pass.xml]: remove margin

File: app/src/main/res/layout/fragment_boarding_pass.xml [hunk 4]
```diff
@@ -0,0 +1,313 @@
...
+                android:layout_toLeftOf="@+id/boardingPassDepartDetail"
+                android:layout_alignParentLeft="true"
+                android:layout_alignParentStart="true"
+                android:id="@+id/boardingPassInfoGate" />
+
+        <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textAppearance="?android:attr/textAppearanceSmall"
+                android:layout_marginLeft="5dp"
```
Comment (feresr) [inline app/src/main/res/layout/fragment_boarding_pass.xml]: remove margin

File: app/src/main/res/layout/fragment_boarding_pass.xml (hunk 1 above)
Comment (Roluisker) [inline app/src/main/res/layout/fragment_boarding_pass.xml]: True!, fixed :)

File: app/src/main/res/layout/fragment_boarding_pass.xml (hunk 2 above)
Comment (Roluisker) [inline app/src/main/res/layout/fragment_boarding_pass.xml]: ready.

**Pull request merged**